mobile/e2e-tests/scripts/live_word_monitor.py

```
@@ -625,28 +1133,332 @@ def finalize_utterance(self, utterance: UtteranceState, now_ms: int) -> None:
+                    "recovered_average_delay_ms": average_delay_ms,
+                },
+            )
+
+    def get_audio_output_probe(self, now_ms: int, refresh_interval_ms: int = 2000) -> tuple[str | None, str | None]:
+        if self.last_audio_output_check_ts_ms and now_ms - self.last_audio_output_check_ts_ms < refresh_interval_ms:
+            return self.last_audio_output_device_name, self.last_audio_output_check_error
+
+        try:
+            self.last_audio_output_device_name = get_default_output_device_name()
+            self.last_audio_output_check_error = None
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)